MTGToolbox.Abstract/MTGToolbox.Abstract.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I also moved everything except the web project to NET Standard because there's no need for the Core SDK.